fairlearn/metrics/_balanced_root_mean_squared_error.py

```diff
@@ -41,6 +41,6 @@ def _balanced_root_mean_squared_error(y_true, y_pred, sample_weight=None):
         y_ta_s = y_ta[indices]
         y_pa_s = y_pa[indices]
         s_w_s = s_w[indices]
-        errs[i] = math.sqrt(skm.mean_squared_error(y_ta_s, y_pa_s, s_w_s))
+        errs[i] = math.sqrt(skm.mean_squared_error(y_ta_s, y_pa_s, sample_weight=s_w_s))
```

Comment:
We had a lurking bug?

Reply:
Not a bug, but sklearn is deprecating certain positional args and making them keyword only. I saw the warnings and fixed it right away. I'm sure you don't mind :-)

Reply:
Not at all :-)